sdk/src/connections/network/network.cpp

```diff
@@ -425,18 +428,40 @@ int Network::callback_function(struct lws *wsi,
                 in = static_cast<void *>(clientData->data.data());
                 len = clientData->data.size();
             }
+            if (static_cast<unsigned char *>(in)[0] == '0') {
+                // process message without frame
+                google::protobuf::io::ArrayInputStream ais(
+                    static_cast<char *>(in) + 1, static_cast<int>(len) - 1);
+                CodedInputStream coded_input(&ais);
+                recv_buff[connectionId].ParseFromCodedStream(&coded_input);
+
+                recv_data_error = 0;
+                Data_Received[connectionId] = true;
+
+                /*Notify the host SDK that data is received from server*/
+                Cond_Var[connectionId].notify_all();
+            } else {
+                // process message of only frame data
+                uint8_t *m_frame = NULL;
+                int m_frameLength;
+                m_frameLength = static_cast<int>(len) - 1;
+
+                if (m_frame != NULL) {
```

Comment:
you always set m_frame to NULL (just 4 lines above). This if() statement doesn't really make sense.

Reply:
removed (not needed anymore)